redis/src/cluster_slotmap.rs

```diff
@@ -94,17 +101,26 @@ impl SlotMap {
                 SlotMapValue {
                     addrs: shard_addrs_arc.clone(),
                     start: slot.start,
-                    latest_used_replica: AtomicUsize::new(0),
+                    last_used_replica: AtomicUsize::new(0),
                 },
             );
         }
 
         slot_map
     }
 
+    #[allow(dead_code)] // used in tests
```

Comment:
nit:  "tests" (when placed on the same file) are allowed to access private members, so the test can access the property member directly